R/appender.R

```diff
@@ -97,3 +109,29 @@ appender.tee <- function(file){
     cat(line, file=file, append=TRUE, sep='')
   }
 }
+
+# Write to a dynamically-named file (and optionally the console)
+appender.file2 <- function(format, console = FALSE,
+                           default.func = "shell", datetime.fmt = "%Y%m%dT%H%M%S"){
```

Comment:
The argument `default.func` needs more explanation. It's not clear to me what the motivation is for replacing `(shell)` with `shell` as a function name.